packages/flame/test/effects2/move_effect_test.dart

```diff
@@ -17,7 +17,7 @@ void main() {
       game.update(0);
 
       object.add(
```

Comment:
Not related to this PR, but we should call the `object` `component` like in the rest of the tests.
Also `flameGame.test` can be used here.